### PR TITLE
ci: add some automation

### LINKED ISSUE #15: Clean up the flake
There's way too much stuff in it left over from Lyceum that we don't actually need here.

### PR DESCRIPTION
**Pull request by schonfinkel**

### CODE REVIEW

**Comment by schonfinkel** [comment]
@z-silver can you help me out with this later? the pure build (`nix build`) is almost working

File: flake.nix [hunk 1]
```diff
@@ -9,123 +14,183 @@
...
-        ERLANG_PATH = "${erlangLatest}";
-      };
-    in
+        version = "0.0.0";
+
+        # Erlang
+        erlangLatest = pkgs.erlang_27;
+        erlangLibs = getErlangLibs erlangLatest;
+
+        # Zig shit (Incomplete)
```
Comment (z-silver) [inline flake.nix]:
```suggestion
        # Zig
```

**Review by z-silver — CHANGES_REQUESTED** [review]
Even if the build doesn't quite work, this is certainly an improvement over what we currently have.

**Review by z-silver — APPROVED** [review]
Thank you.

**Pull request merged**